pkg/devfile/sample_test.go

```diff
@@ -26,20 +26,20 @@ func TestGetRegistrySamples(t *testing.T) {
 	}{
 		{
 			name:     "Fetch the sample placeholder",
-			registry: "sample-placeholder",
+			registry: DEVFILE_REGISTRY_URL,
```

Comment:
we dont want to ping the actual registry for tests and spam the telemetry. https://registry.stage.devfile.io would be preferred here